src/cupbearer/data/_shared.py

```diff
@@ -75,7 +34,9 @@ def get_transforms(self) -> list[Transform]:
         processing to this that can't be handled in __post_init__ (see BackdoorData
         for an example).
         """
-        return list(self.transforms.values())
+        if self.no_augmentation:
```

Comment:
I think I lean towards not making any changes to `DatasetConfig`, i.e. just stick with `transforms`. Then in `PytorchConfig`, we could have a `default_augmentations = True` field, and if this is set, we add the default augmentations to `transforms` inside `__post_init__`. I think that way the complexity would be only in `PytorchConfig`, where it's actually needed, and `DatasetConfig` and `BackdoorData` wouldn't even need to know about augmentations.

Reply:
Sounds good. I think I first believed I needed the extra control of when the augmentations were added but, as it looks now it can probably be simplified like this.

Reply:
There seems to be a bug when configs are saved to yaml. In config.yaml, transforms are listed in alphabetical order.